x-pack/plugins/fleet/server/saved_objects/security_solution/to_v7_11_0.ts

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
is this file meant to be a new file, or a rename?

Reply:
Whoops! Thanks for spotting this. I have it fixed locally and will push shortly.